sematic/ui/packages/common/src/layout/ThreeColumns.tsx

```diff
@@ -0,0 +1,52 @@
+import { useMemo } from 'react';
+import styled from '@emotion/styled';
+import theme from 'src/theme/new';
+
+const Left = styled.div`
+    min-width: 300px;
+    width: 19.841%;
+    height: 100%;
+    border-right: 1px solid ${theme.palette.p3border.main};
+    padding: 12px 25px;
```

Comment:
The top and bottom padding should be set on each section so that they are consistent and can be adjusted centrally.
So this should be only setting the left and right padding (25px).